server

```diff
@@ -127,6 +127,17 @@ query_default_version() {
   DEFAULT_SCYLLA_VERSION=$(echo $DEFAULT_SCYLLA_VERSION_RAW | sed -e "s/.*version\":\"\(.*\)\".*/\1/g")
 }
 
+packages_update () {
+  OS_TYPE=$(cat /etc/os-release | awk -F '=' '/^NAME/{print $2}' | awk '{print $1}' | tr -d '"')
+  if [ "$OS_TYPE" == "Ubuntu" ]; then
```

Comment:
following the comment above you can change this line to 
```
if [ "$NAME" == "Ubuntu" ] ; then
```

Reply:
Done. Fixed.